payment.js

```diff
@@ -0,0 +1,67 @@
+// @ts-nocheck
+import * as React from 'react';
+import { Component } from 'react';
+import { Button, Platform, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { CFPaymentGatewayService, } from 'react-native-cashfree-pg-sdk';
+import { CFDropCheckoutPayment, CFEnvironment, CFPaymentComponentBuilder, CFPaymentModes, CFSession, CFThemeBuilder, } from 'cashfree-pg-api-contract';
+import CommonStyles from '../../globals/CommonStyles';
+import showAlertWithMessage from '../../globals/AlertComponent';
+import style from './style';
+
+const Payment = () => {
+    
+    onVerify = (orderID) =>  {
+      showAlertWithMessage('Payment has been processed successfully!');
+    }
+    onError = (error) => {
+      console.error(error);
+      const message = error?.message ? error.message : "Payment Failed"; 
+      showAlertWithMessage(message);
+    }
+    
+    React.useEffect(()=> {
+      CFPaymentGatewayService.setCallback(this);
+      return () => {
+        CFPaymentGatewayService.removeCallback()
+      }
+    })
```

Comment:
Are you intentionally removed second param (dependency array []) from useEffect? This will be called and executed all re-renders instead of mount and unmount.